.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!-- 🎉⛓🎉 Thank you for the PR!!! 🎉⛓🎉 -->
+
+# Changes
+
+<!-- Describe your changes here- ideally you can get that description straight from your descriptive commit message(s)! -->
+
+# Submitter Checklist
+
+As the author of this PR, please check off the items in this checklist:
+
+- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
+- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
+- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
+- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
+  functionality, content, code)
+- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
```

Comment:
You can omit this. We don't track kind labels in this repo (not that we're against them - we just haven't had any use for them so far).

Reply:
Done.